netkan/netkan/scheduler.py

```diff
@@ -48,6 +49,52 @@ def schedule_all_netkans(self):
         for batch in sqs_batch_entries(messages):
             self.client.send_message_batch(**self.sqs_batch_attrs(batch))
 
+    def cpu_credits(self, cloudwatch, instance_id, start, end):
+        stats = cloudwatch.get_metric_statistics(
+            Dimensions=[{'Name': 'InstanceId', 'Value': instance_id}],
+            MetricName='CPUCreditBalance',
+            Namespace='AWS/EC2',
+            StartTime=start.strftime("%Y-%m-%dT%H:%MZ"),
+            EndTime=end.strftime("%Y-%m-%dT%H:%MZ"),
+            Period=10,
+            Statistics=['Average'],
+        )
+        # A pass consumes around 40 credits, with an accrue rate of 24/hr.
```

Comment:
Is this estimate (40 cred per pass) still up to date with current performance after KSP-CKAN/CKAN#2940 ? Not crucial to update it, but if someone does need the info, it'd be nice to have it accurate.

Reply:
I've made some adjustments here. Lemme know what you think :slightly_smiling_face:

Reply:
Looks like a helpful update! Thanks!